assets/js/theme/common/faceted-search.js

```diff
@@ -429,19 +427,6 @@ class FacetedSearch {
             this.collapsedFacets = _.without(this.collapsedFacets, id);
         }
     }
-
-    onPopState() {
```

Comment:
@VladDlogush do we really have to remove this code?
Please check if the user will be able to use the browser's "Forward ->" and " <- Back" navigation when navigating through the pagination without the `onPopState` method.

Reply:
@bc-yevhenii-buliuk Everything works correctly, you can see it in the screen recording

Reply:
@bc-vlad-dlogush could you clarify at which second of your recording you use browser navigation?

Reply:
Hmm, i see, thanks

Reply:
@bc-yevhenii-buliuk fixed, browser navigation works correctly!